frontend/src/metabase/dashboard/components/DashCard/DashCard.tsx

```diff
@@ -88,6 +88,10 @@ export interface DashCardProps {
   onChangeLocation: (location: LocationDescriptor) => void;
 
   downloadsEnabled: boolean;
+
+  /* Auto-scroll to this card on mount */
+  autoScroll: boolean;
+  reportAutoScrolledToDashcard: () => void;
```

Comment:
Maybe a comment on this prop? It's tricky to parse

Reply:
I wasn't sure right away whether this function reports something to a dashcard or reports that the page autoscrolled to a dashcard. I don't think you need to change the variable name but a jsdoc prose explanation would be handy.